chronosphere/tfschema/log_allocation_config.go

```diff
@@ -38,12 +50,24 @@ var LogAllocationConfigSchema = &schema.Schema{
 	Required: true,
 }
 
-var LogDatasetAllocationSchema = &schema.Resource{
-	Schema: map[string]*schema.Schema{
-		"dataset_slug": {
-			Type:     schema.TypeString,
-			Required: true,
+var LogPrioritiesSchema = &schema.Schema{
+	Type: schema.TypeList,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"high_priority_filters": {
```

Comment:
nit: should we use a singular name here and for low_priority since these are blocks?

when there's multiple, it's not a list, but repeated blocks like:
```
      high_priority_filters {
        ...
      }
      high_priority_filters {
        ...
      }
```

I think that would be clearer with singular names:
```
      high_priority_filter {
        ...
      }
      high_priority_filter {
        ...
      }
```